### DISCUSSION TITLE
Are images supposed to be displayed without any sort of anti-aliasing?

### DISCUSSION THREAD

**Discussion by alexhx5** [d2phap/ImageGlass#283] [General] [answered]
### Description 
ImageGlass displays images without any anti-aliasing, all the straight lines are aliased on all the images. 

- Is this intentional?
- If it's caused by the lack of anti-aliasing is there a chance you can add an option to turn on anti-aliasing?

### System info:
- ImageGlass version: 4.5.11.27
- OS version: Win 10
- Display Resolution & DPI: 1080p | 125% DPI

### Preview comparison
I've made the screenshots of the open programs displaying images so you can see what I'm seeing:

- Open both images in the new tab
- Go back and forth with `Ctrl + PageUp` and  `Ctrl + PgDown`

![1](https://user-images.githubusercontent.com/13764106/33808093-914ec330-ddf1-11e7-9267-32b302ca7d4c.png)
![2](https://user-images.githubusercontent.com/13764106/33808094-9167ab70-ddf1-11e7-941e-30ba8f907ffb.png)

**Comment by d2phap** [comment, marked as answer]
Hi @AlexAlabyev 

There is an option in Settings dialog called `Zoom optimization` where you can change it to `Smooth pixels`.

![image](https://user-images.githubusercontent.com/3154213/33815899-1ab950ee-de70-11e7-9f80-1f7a1adceabe.png)

**Comment by alexhx5** [comment]
@d2phap thanks, that works! Makes it quite laggy to pan around when you zoom in, but it works.
